cardano-cli/src/Cardano/CLI/EraBased/Run/Governance/DRep.hs

```diff
@@ -69,21 +71,23 @@ runGovernanceDRepIdCmd _ vkOrFp idOutputFormat mOutFile = do
 -- Registration Certificate related
 
 runGovernanceRegistrationCertificateCmd
-  :: ConwayEraOnwards era
+  :: forall era ledgerera. ()
```

Comment:
So we can simplify this type signature as follows:
```
runGovernanceRegistrationCertificateCmd
  :: ConwayEraOnwards era
  -> VerificationKeyOrHashOrFile DRepKey
  -> Lovelace
  -> Maybe (Ledger.Anchor (Ledger.EraCrypto (ShelleyLedgerEra era)))
  -> File () Out
  -> ExceptT RegistrationError IO ()
runGovernanceRegistrationCertificateCmd ...
```
We should also be able to remove the `TypeOperators` extension.